Source/WebCore/rendering/RenderFileUploadControl.cpp

```diff
@@ -46,18 +46,22 @@ using namespace HTMLNames;
 
 WTF_MAKE_ISO_ALLOCATED_IMPL(RenderFileUploadControl);
 
-const int afterButtonSpacing = 4;
+constexpr int afterButtonSpacing = 4;
+constexpr int buttonShadowHeight = 2;
+
+#if !PLATFORM(COCOA)
+constexpr int iconHeight = 16;
+constexpr int iconWidth = 16;
+#endif
+
 #if !PLATFORM(IOS_FAMILY)
-const int iconHeight = 16;
-const int iconWidth = 16;
-const int iconFilenameSpacing = 2;
-const int defaultWidthNumChars = 34;
+constexpr int iconFilenameSpacing = 2;
+constexpr int defaultWidthNumChars = 34;
 #else
 // On iOS the icon height matches the button height, to maximize the icon size.
-const int iconFilenameSpacing = afterButtonSpacing;
-const int defaultWidthNumChars = 38;
+constexpr int iconFilenameSpacing = afterButtonSpacing;
+constexpr int defaultWidthNumChars = 38;
```

Comment:
I think this would be easier to read if it was structured as:

```
#if PLATFORM(COCOA)
#if PLATFORM(IOS_FAMILY)
...
#else
...
#endif 
#else
...
#endif
```

Reply:
@pxlcoder that would result in
```
#if PLATFORM(COCOA)
#if PLATFORM(IOS_FAMILY)
// On iOS the icon height matches the button height, to maximize the icon size.
constexpr int iconFilenameSpacing = afterButtonSpacing;
constexpr int defaultWidthNumChars = 38;
#else
constexpr int iconFilenameSpacing = 2;
constexpr int defaultWidthNumChars = 34;
#endif
#else
constexpr int iconHeight = 16;
constexpr int iconWidth = 16;
constexpr int iconFilenameSpacing = 2;
constexpr int defaultWidthNumChars = 34;
#endif
```

which I personally think is more confusing, plus it adds some duplication

Reply:
Sounds good. Let's just move the comment "// On iOS the icon height matches the button height, to maximize the icon size." underneath line 52, and reword as "On Cocoa platforms...", since it's out of place now.